app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputRadioGroup.scala.html

```diff
@@ -51,6 +51,8 @@ <h1 class="heading-large">
         @if(elements.args.get('_hintText).isDefined) {
             <p class="form-hint">@elements.args.get('_hintText)</p>
         }
+        @hintHtml.map(html => html)
```

Comment:
The result of this will be still `Option[Html]` So here you can leave `@hintHtml`.

Or you can do something like `@hintHtml.getOrElse(something empty here)`

Reply:
You are right of couse.  The mapping does nothing.  I think I meant to do 'forEach'.   But I never knew that twirl (I presume) copes with an Option[Html].